Remove docblocks in sortable trait

a-komarev opened the pull request:
One more minor fix :smile:

freekmurze commented:
Thanks for this, but I'll just remove the entire docblock. Function signature is clear enough.

a-komarev commented:
Okay. I've removed it. And removed docblock from `determineOrderColumnName` because by your logic it's self described too. And resorted methods ordering (protected method `determineOrderColumnName` goes after all public ones).

freekmurze commented:
I've done this myself in the new major version. Thanks though!
